docs/resources/slo.md

```diff
@@ -37,7 +37,11 @@ The following arguments are supported:
 
 * `name` - (Required) The name of the SLO.
 * `description` - (Optional) A description of the SLO's intent and context.
-* `dataset` - (Required) The dataset this SLO is created in. Must be the same dataset as the SLI unless the SLI's dataset is `"__all__"`.
+* `dataset` - (Deprecated, conflicts with `datasets`) The dataset this SLO is created in. Must be the same dataset as the SLI unless the SLI's dataset is `"__all__"`.
+  * If `dataset` is not configured, `datasets` must be configured. 
+* `datasets` - (Required, conflicts with `dataset`) Array of datasets the SLO is evaluated on.
+  * If `datasets` is not configured, `dataset` must be configured.
+  * Must be >= 1 and <= 10. 
```

Comment:
nit: this nested-bullet formatting is not used elsewhere in the docs, so I'd rather we be consistent. Take a look at the trigger docs for an example listing conflicts and details like "up to 10"